### PR TITLE
chore(marge-bot): update and add config parameters

### PR DESCRIPTION
**Pull request by antoinerabany**

### CODE REVIEW

File: charts/marge-bot/templates/deployment.yaml [hunk 1]
```diff
@@ -42,6 +42,16 @@ spec:
                 secretKeyRef:
                   name: {{ include "marge.fullname" . }}-secret
                   key: auth-token
+            - name: MARGE_REBASE_REMOTELY
+              valueFrom:
+                secretKeyRef:
+                  name: { { include "marge.fullname" . } }-secret
```
Comment (spnngl) [inline charts/marge-bot/templates/deployment.yaml:48]: No space between the '{{' and '}}'

File: charts/marge-bot/templates/secret.yaml [hunk 2]
```diff
@@ -7,3 +7,5 @@ data:
   gitlab-url: {{ .Values.secret.gitlabUrl | b64enc | quote }}
   ssh-key: {{ .Values.secret.sshKey | quote }}
   auth-token: {{ .Values.secret.authToken | b64enc | quote }}
+  rebase-remotely: {{ .Values.secret.rebaseRemotely | b64enc | quote }}
```
Comment (spnngl) [inline charts/marge-bot/templates/secret.yaml:10]: Add default values inside values.yml

File: charts/marge-bot/Chart.yaml
Comment (spnngl) [inline charts/marge-bot/Chart.yaml:1]: Increment the chart version

File: charts/marge-bot/values.yaml [hunk 3]
```diff
@@ -40,3 +40,5 @@ secret:
   # yours: base64 -w0 <my_private_key>
   sshKey: ""
   authToken: ""
+  rebaseRemotely: false
```
Comment (spnngl) [inline charts/marge-bot/values.yaml:43]: Are you sure boolean values are supported in marge-bot ?
Seems to use configargparse.ArgParser, don't know if it supports this

Comment (antoinerabany) [reply charts/marge-bot/values.yaml]: right don't knwo either but I guess string is safer

**Pull request merged**